pkg/aws/cloudfront/options.go

```diff
@@ -0,0 +1,21 @@
+package cloudfront
+
+import (
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws/credentials"
+)
+
+type Option func(c *Client)
+
+func WithStaticCredentials(id string, secret string, token string) Option {
```

Comment:
Is `token` required when we configure the service with an AccessId and Secret pair?

Reply:
Looks like token is not necessary and can be left empty `""`. Thanks. Will fix.